webapp/apps/btax/urls.py

```diff
@@ -4,10 +4,10 @@
 
 
 urlpatterns = patterns('',
-    url(r'^$', btax_results, name='tax_form'),
-    url(r'^(?P<pk>\d+)/output.csv/$', csv_output, name='csv_output'),
-    url(r'^(?P<pk>\d+)/input.csv/$', csv_input, name='csv_input'),
-    url(r'^(?P<pk>\d+)/', output_detail, name='output_detail'),
+    url(r'^$', btax_results, name='btax_tax_form'),
+    url(r'^(?P<pk>\d+)/output.csv/$', csv_output, name='btax_csv_output'),
```

Comment:
yes, if you set up all of the `urls.py` correctly, someone would do ospc.org/taxbrain/342 and get the OutputURL with that ID, This has a key to the model that holds user input and results. In this case, we probably want to set up \<host\>/btax/\<number\> for the results page, at least for now.